### PR TITLE
POL-98 - Azure Migrate

### PR DESCRIPTION
**Pull request by dfrankel33**
### Description

Initial release of the RISC/Azure Migrate Integration

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: operational/azure/azure_migrate/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+This policy has the following input parameters required when launching the policy.
+
+- *RISC User ID* - Email address of the RISC User Account which will be used for authentication
+- *RISC Hashed Password* - Hashed password to be used for authentication
+- *RISC Assessment Code* - RISC Assessment Code to be used for authentication
+- *Azure Tenant ID* - the Azure AD Tenant ID used for the Azure API Authentication
+- *Azure Subscription ID* - the Azure Subscription ID used for the Azure API Authentication
+- *Azure Migrate Project Name* - The resource name of the Azure Migrate Project where RISC data should be populated
+
```
Comment (cdwilhelm) [inline operational/azure/azure_migrate/README.md]: Include Policy Actions section

File: operational/azure/azure_migrate/risc-azure-migrate.pt [hunk 2]
```diff
@@ -0,0 +1,309 @@
+name "Azure Migrate Integration"
+rs_pt_ver 20180301
+type "policy"
+short_description "This Policy will collect the resources from a RISC Foundations assessment and seed Azure Migrate with the discovered servers. See the [README](https://github.com/rightscale/policy_templates/tree/master/operational/azure/azure_migrate) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```
Comment (cdwilhelm) [inline operational/azure/azure_migrate/risc-azure-migrate.pt]: include details about action

File: operational/azure/azure_migrate/risc-azure-migrate.pt [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+    }
+
+  EOS
+end
+
+policy "discovered_devices" do
+  validate $ds_riscgetAssetsByStack do
+    summary_template "Discovered Servers to be Added to Azure Migrate"
+    detail_template <<-EOS
+# Discovered Servers to be Added to Azure Migrate
```
Comment (cdwilhelm) [inline operational/azure/azure_migrate/risc-azure-migrate.pt]: Is a new line needed to parse the markdown?

File: operational/azure/azure_migrate/risc-azure-migrate.pt [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+{{ range data -}}
+| {{.hostname}} | {{.identifying_ip}}
+{{ end -}}
+EOS
+    escalate $esc_azure_migrate
+    check false
+  end
+end
+
+escalation "esc_azure_migrate" do
```
Comment (cdwilhelm) [inline operational/azure/azure_migrate/risc-azure-migrate.pt]: approval should be added

File: operational/azure/azure_migrate/risc-azure-migrate.pt (hunk 2 above)
Comment (dfrankel33) [reply operational/azure/azure_migrate/risc-azure-migrate.pt]: that's what "seed Azure Migrate with the discovered servers" is

File: operational/azure/azure_migrate/risc-azure-migrate.pt (hunk 3 above)
Comment (dfrankel33) [reply operational/azure/azure_migrate/risc-azure-migrate.pt]: Nope.  the incident renders properly

File: operational/azure/azure_migrate/risc-azure-migrate.pt (hunk 4 above)
Comment (dfrankel33) [reply operational/azure/azure_migrate/risc-azure-migrate.pt]: the purpose of this policy is really not to alert.  it's an integration whose sole purpose is to push all data from a RISC Foundations Assessment to Azure Migrate for migration planning.  approval defeats the purpose.

**Pull request merged**